### PR TITLE
fix spine will crash when load/unload spine res frequently

### PR DESCRIPTION
**Pull request by random634**
load spine res  ref count = 1
SpineRenderer1 hold ref +1 = 2
play with spine happy
unload spine res ref -1 = 1

load spine res ref count (not change) = 1
SpineRenderer2 hold ref +1 = 2
play with spine happy
unload spine res ref -1 = 1

load spine res ref count (not change) = 1
SpineRenderer3 hold ref +1 = 2
play with spine happy

SE gc
SpineRenderer1 release ref -1 = 1
SpineRenderer2 release ref -1 = 0
play with spine unhappy (crashed)

### CODE REVIEW

File: cocos/scripting/js-bindings/manual/jsb_spine_manual.cpp [hunk 1]
```diff
@@ -406,7 +406,10 @@ static bool js_register_spine_initSkeletonData (se::State& s)
     
     auto mgr = spine::SkeletonDataMgr::getInstance();
     bool hasSkeletonData = mgr->hasSkeletonData(uuid);
-    if (hasSkeletonData) return true;
+    if (hasSkeletonData) {
+      mgr->retainByUUID(uuid)
+      return true;
```
Comment (sunnylanwanjun) [inline cocos/scripting/js-bindings/manual/jsb_spine_manual.cpp]: 麻烦把缩进改为四个空格，谢谢。

**Comment by sunnylanwanjun** [comment]
麻烦把合并分支改为2.1.2

**Comment by random634** [comment]
@sunnylanwanjun 改好了

**Comment by sunnylanwanjun** [comment]
我把你的修改cherry pick过来了，提了个新的pr
https://github.com/cocos-creator/cocos2d-x-lite/pull/1758

**Comment by jareguo** [comment]
感谢反馈，我们重新提了一个 PR，进入到 2.1 分支。

**Pull request closed without merging**